src/Contas/ContaCorrente.java

```diff
@@ -16,7 +16,7 @@ void setLimite() {
 
     @Override //ITaxas()
     public double calculaTaxas(){
-        return this.dono.getClass() == ClientePessoaFisica.class ? 10:20;
+        return dono instanceof ClientePessoaFisica ? 10:20;
```

Comment:
Vitor, essa implementação funciona, mas não é a ideal porque não segue as regras do polimorfismo. Polimorfismo permite tratar objetos de diferentes classes de maneira uniforme, sem precisar saber o tipo específico do objeto. Portanto, é necessário acessar diretamente o método calcularTaxas do dono.
Assim: return this.getDono().calcularTaxas().